### PR TITLE
DVR Compatibility 📼

### LINKED ISSUE #18: DVR's cassettes compatibility
We need to check if Vinyl could use DVR's cassettes out of the box. For example I currently have around 20 cassettes (json files), Vinyl should be able to just use them.

### PR DESCRIPTION
**Pull request by RuiAAPeres**
closes #18

### CODE REVIEW

File: Vinyl/Extensions/Response.swift [hunk 1]
```diff
@@ -14,11 +14,11 @@ extension Response {
         guard
             let urlString = encodedResponse["url"] as? String,
             let url =  NSURL(string: urlString),
-            let statusCode = encodedResponse["statusCode"] as? Int,
```
Comment (dmcrodrigues) [inline Vinyl/Extensions/Response.swift]: Can't we support both? I rather use our approach/naming which I find more meaningful :disappointed:

**Comment by dmcrodrigues** [comment]
:guitar: :notes:

File: Vinyl/Extensions/Response.swift (hunk 1 above)
Comment (RuiAAPeres) [inline Vinyl/Extensions/Response.swift]: It would do more harm than good to be honest. :/

Comment (dmcrodrigues) [inline Vinyl/Extensions/Response.swift]: Good point :+1:

**Comment by dmcrodrigues** [comment]
:bike: :notes:

**Pull request merged**